sshspawner/sshspawner.py

```diff
@@ -3,29 +3,35 @@
 import shlex
 from textwrap import dedent
 import warnings
+import random
 
-from traitlets import Bool, Unicode, Integer
+from traitlets import Bool, Unicode, Integer, List, observe
 
 from jupyterhub.spawner import Spawner
 
 
 class SSHSpawner(Spawner):
-    
+
+    # http://traitlets.readthedocs.io/en/stable/migration.html#separation-of-metadata-and-keyword-arguments-in-traittype-contructors
+    # config is an unrecognized keyword
+
+    nodes = List(trait=Unicode(), default_value=['Cori19','Cori20'],
```

Comment:
Please rename to `remote_hosts` for symmetry with `remote_host`, and remove `default_value` argument.